### PR TITLE
Update Rust Boilerplate tutorial

### PR DESCRIPTION
**Pull request by yogh333**

### CODE REVIEW

File: pages/docs/device-app/develop/tutorials/rust.mdx [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
-  production environment.
+<Callout type="info">
+Stax support is not ready yet and should be available later.
 </Callout>
 
-The `Rust` Boilerplate application is taylored to showcase both the capabilities of the Ledger Secure `Rust` SDK and our
-development environment and tools.
+The `Rust` Boilerplate application is taylored to showcase both the capabilities of the Ledger Device `Rust` [SDK](https://github.com/LedgerHQ/ledger-device-rust-sdk/tree/master) 
+(also published on [crates.io](https://crates.io/crates/ledger_device_sdk)) and our development environment ([Docker containers](https://github.com/LedgerHQ/ledger-app-builder), 
+[device emulator](https://github.com/LedgerHQ/speculos), [test framework](https://github.com/LedgerHQ/ragger)).
```
Comment (tdejoigny-ledger) [inline pages/docs/device-app/develop/tutorials/rust.mdx:11]:
```
([Docker containers](https://github.com/LedgerHQ/ledger-app-builder), 
[device emulator](https://github.com/LedgerHQ/speculos), [test framework](https://github.com/LedgerHQ/ragger)).
```

this part has already been listed in other parts of the dev portal (could be removed)

File: pages/docs/device-app/develop/tutorials/rust.mdx [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
 
+# The `Rust` 🦀 boilerplate application
 
-# The `Rust` boilerplate application
-
-<Callout type="warning">
-  This section is still a Work In Progress, as the `Rust` SDK is not yet stable and fully integrated in our
-  production environment.
+<Callout type="info">
+Stax support is not ready yet and should be available later.
```
Comment (tdejoigny-ledger) [inline pages/docs/device-app/develop/tutorials/rust.mdx:6]:
```suggestion
Stax device support is not ready yet and should be available later.
```

File: pages/docs/device-app/develop/tutorials/rust.mdx (hunk 1 above)
Comment (yogh333) [reply pages/docs/device-app/develop/tutorials/rust.mdx:11]: Yes maybe it is redundant a little bit but I think it is still useful to provide those links here as well.

**Review by tdejoigny-ledger — APPROVED** [review]

File: pages/docs/device-app/develop/tutorials/rust.mdx [hunk 3]
```diff
@@ -1,20 +1,33 @@
...
 
 [Click here](https://github.com/LedgerHQ/app-boilerplate-rust) to fork the repository and start your project.
+
+## Continuous Integration (CI)
+
+In the Rust boilerplate application repository you will find [a list of workflows](https://github.com/LedgerHQ/app-boilerplate-rust/tree/main/.github/workflows) 
+to set up an efficient CI pipeline for your project.
+
+Three of them are <u>mandatory</u> to submit your device application for review:
+- [Coding Style Check](https://github.com/LedgerHQ/app-boilerplate-rust/blob/main/.github/workflows/coding_style_checks.yml):
```
Comment (tdejoigny-ledger) [inline pages/docs/device-app/develop/tutorials/rust.mdx:26]: not sure that this one should be mandatory

**Pull request merged**